tools/fast/tests/retrieval_test.go

```diff
@@ -58,7 +58,6 @@ func TestRetrieval(t *testing.T) {
 	options := make(map[string]string)
 	options[localplugin.AttrLogJSON] = "0"                               // Disable JSON logs
 	options[localplugin.AttrLogLevel] = "4"                              // Set log level to Info
-	options[localplugin.AttrUseSmallSectors] = "true"                    // Enable small sectors
```

Comment:
I don't see a change to the `EnvironmentMemoryGenesis`, I assume this doesn't build at the moment.

Reply:
I have added the appropriate argument to `NewEnvironmentMemoryGenesis`.